### PR TITLE
Added details of how to get a Forms license that will work in FIPS-compliant environments.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added details of how to get a Forms license that will work in FIPS-compliant environments.

Documents a solution we've come up with from a customer case.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [X] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 13+.

## Deadline (if relevant)

Can be reviewed and published at any time.

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @AndyButland thank you for the PR 🙌 

We will take a look at it as soon as we can 😊

File: 13/umbraco-forms/installation/the-licensing-model.md [hunk 1]
```diff
@@ -105,3 +105,22 @@ The value contains the path of your custom license directory relative to the roo
 {% hint style="warning" %}
 This will also change the location for other Umbraco-related licenses in this project.
 {% endhint %}
+
+## Note on Federal Information Processing Standards (FIPS) Compliant Environments
+
+The algorithm used to decrypt Forms licenses is not supported on locked down FIPS compliant environments, such as those used in the defense industry.
+
+If you are in this situation and unable to resolve via configuration of the environment, please reach out to Umbraco support.
```
Comment (eshanrnh) [inline 13/umbraco-forms/installation/the-licensing-model.md:113]:
```suggestion
If you are in this situation and unable to resolve it via configuration of the environment, please reach out to Umbraco support.
```

File: 13/umbraco-forms/installation/the-licensing-model.md [hunk 2]
```diff
@@ -105,3 +105,22 @@ The value contains the path of your custom license directory relative to the roo
...
+
+We have the possibility of generating and providing Forms licenses using alternate algorithms.
+
+These can be used if you reference `Umbraco.Licensing` version `13.0.1` or higher.
+
+And apply the following configuration with the appropriate algorithm - `DES` (the default), `TripleDES` or `AES`:
```
Comment (eshanrnh) [inline 13/umbraco-forms/installation/the-licensing-model.md:119]:
```suggestion
Apply the following configuration with the appropriate algorithm - `DES` (the default), `TripleDES`, or `AES`:
```

File: 13/umbraco-forms/installation/the-licensing-model.md [hunk 3]
```diff
@@ -105,3 +105,22 @@ The value contains the path of your custom license directory relative to the roo
...
+
+## Note on Federal Information Processing Standards (FIPS) Compliant Environments
+
+The algorithm used to decrypt Forms licenses is not supported on locked down FIPS compliant environments, such as those used in the defense industry.
+
+If you are in this situation and unable to resolve via configuration of the environment, please reach out to Umbraco support.
+
+We have the possibility of generating and providing Forms licenses using alternate algorithms.
+
+These can be used if you reference `Umbraco.Licensing` version `13.0.1` or higher.
```
Comment (eshanrnh) [inline 13/umbraco-forms/installation/the-licensing-model.md:117]:
```suggestion
{% hint style="info" %}
You can use this configuration if you're referring to `Umbraco.Licensing` version `13.0.1` or higher.
{% endhint %}
```

File: 13/umbraco-forms/installation/the-licensing-model.md [hunk 4]
```diff
@@ -105,3 +105,22 @@ The value contains the path of your custom license directory relative to the roo
 {% hint style="warning" %}
 This will also change the location for other Umbraco-related licenses in this project.
 {% endhint %}
+
+## Note on Federal Information Processing Standards (FIPS) Compliant Environments
```
Comment (eshanrnh) [inline 13/umbraco-forms/installation/the-licensing-model.md:109]:
```suggestion
## Federal Information Processing Standards (FIPS) Compliant Environments
```

**Review by eshanrnh — COMMENTED** [review]
I've made some minor suggestions, @AndyButland. Thanks for the PR 💪

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Applied the suggestions from Esha, so am merging this in 🙌

**Pull request merged**